packages/cli-lib/lang/en.lyaml

```diff
@@ -2,7 +2,7 @@ en:
   cli:
     commands:
       generalErrors:
-        srcIsProject: "\"{{ src }}\" is a project folder. Please use a command with the \"hs project\" prefix."
+        srcIsProject: "\"{{ src }}\" is a project folder. Did you mean \"hs project {{command}}\"?"
```

Comment:
I was playing around with this a little bit, and I think we could improve the wording a bit more. The `src` folder won't always be a project folder. The `getIsInProject` util will also return true for a folder that's nested inside of a project.

Maybe something like this is better?
`The path "\"{{ src }}\" is in a project folder. Did you mean \"hs project {{command}}\"?"`
Or maybe just adding "in" is enough?
`"\"{{ src }}\" is in a project folder. Did you mean \"hs project {{command}}\"?"`

Reply:
I like Option 2, a bit simpler and still gets the point across.